LICENSE

```diff
@@ -0,0 +1,42 @@
+The Hippocratic License (MIT - Hippocratic)
+
+Copyright 2015-present Humanity Codes LLC
```

Comment:
Would you mind updating the year to 2020 instead of 2015?

Reply:
@salahuddinlali, would you mind updating the license to reflect the new year?

Reply:
Hello Erik, I have made the change to the file year, please review, thanks!

Reply:
@egillespie for a notification test